install.sh

```diff
@@ -59,10 +43,26 @@ if [ -n "${RPMFUSION_MIRROR}" ]; then
 fi
 
 # run common packages script
-/tmp/packages.sh
+"$BUILDCONTEXT_DIR/packages.sh"
+
+# Handle Kernel Skew with override replace
```

Comment:
The `# Handle Kernel Skew with override replace` section was moved after `# run common packages script`, but we wish to know that any kernel changes cleanly install before moving on to the common packages install.

Please return to the original sequence.

Reply:
Oh, i'm sorry i failed to document the reason. I moved this to "after the common packages" because i observed dracut falling back to pigz instead of the intended zstd if/when regenerating the intiramfs.

Reply:
The dracut run you see doesn't actually get included into the final image when used with kernel-install for some reason. Only when we manually invoke it does it seem to get included correctly.

Reply:
I removed it from this PR anyway - it doesnt belong here.